### ISSUE TITLE
is the `# title` in the SPARQL queries meant to be used by the WDQS?

### ISSUE THREAD

**Issue by egonw** [WDscholia/scholia#2304]
## Context
Multiple SPARQL queries have a title in a comment, e.g. https://github.com/WDscholia/scholia/blob/6d14811f1a52ec68c2b359f64807b306fd82d257/scholia/app/templates/venue_topics.sparql#L1

But many of `# title:` which is not recognized by the WDQS, while `#title:` is (see below screenshots).

The space is [quite prevalent](https://github.com/search?q=repo%3AWDscholia%2Fscholia+%22%23+title%22+language%3ASPARQL&type=code&l=SPARQL).

## Question

Is the use of `#title:` in the queries meant to be used by the WDQS?

## Screenshots

With space:

![image](https://github.com/WDscholia/scholia/assets/26721/f0eef042-c1cc-42ea-b737-075774215d27)

Without space:

![image](https://github.com/WDscholia/scholia/assets/26721/3aff8677-0b2b-4bc1-affc-392afaa0d0f5)

**Comment by Daniel-Mietchen** [comment]
I think these should be without space, so as to give the blue ribbon atop of the results.
Some of those with space might be older than that WDQS feature.

**Comment by egonw** [comment]
Okay, I will go ahead and make a patch then.

**Comment by fnielsen** [comment]
If I remember correctly, I added the "title" before Wikimedia (in Berlin?) started to use the keyword (without the space). I do not think we should use title because the iframes would then get a title banner which might not be visually nice on the scholia page. The "tool" keyword is added automagically. We could suggest that this becomes a Wikimedia keyword.

**Comment by egonw** [comment]
> I do not think we should use title because the iframes would then get a title banner

Good point! I agree with that. We can remove that line when used in the `iframe` context, just like we add the `#tool:` line. Would that be an idea?
